### ISSUE TITLE
Messaging badges to look consistent

### ISSUE THREAD

**Issue by rferromoreno** [foxtrick-ng/foxtrick#33]
## Short description of the problem

I would like to have Foxtrick changing the `ft-new-mail` styles so it can be similar to hattrick's `unread-badge` css class

Current look:
![image](https://github.com/user-attachments/assets/8e21df5e-eae3-4390-ae5e-60b8c5d1338a)

I would like to have something like this:
![image](https://github.com/user-attachments/assets/ce1086a9-60b4-455f-a271-d4001df61d0f)

I think the change is not that hard. What do you think?

**Comment by murko69** [comment]
The number 37 don't even sit by the mailbox - Is that really unread mail?
Looks like those are unread news on the MyOffice page. Screenshot is a bit small to see that though.

I just sent myself a HT-mail and I get a yellow ring (1) .
If I enable the mail-module from Foxtrick I get these two extra:
![Screenshot(3)](https://github.com/user-attachments/assets/a5bdbd06-a553-4594-ba26-869cc34b0df1)

**Comment by rferromoreno** [comment]
The number is a sum of unread messages in My Office and also in Mailbox

![image](https://github.com/user-attachments/assets/93d77e6a-0e83-4851-8d8a-6f8acd8b3e32)

**Comment by murko69** [comment]
Since I don't use the module I can't speak for- nor against anything really,
I know it also has a forum feature (mimic) for new messages and that also sound notifications is a feature.

So what should go, what should stay is up for discussion of course :)

**Comment by rferromoreno** [comment]
> Since I don't use the module I can't speak for- nor against anything really, I know it also has a forum feature (mimic) for new messages and that also sound notifications is a feature.
> 
> So what should go, what should stay is up for discussion of course :)

Yes, but I'm not sure if that is actually working (the visual part).

I think we should leave the notifications as it is, the only thing I'm proposing is just making the other badges to look consistent with the rest of the site

**Comment by unnecessarydave** [comment]
I assume this re-styling of the little badge by Foxtrick made sense at some point in the distant past.  I had a very quick look at the code, and as far as I can see the re-styling does not convey any additional information to the user.

Personally, I can't see any reason why we shouldn't use the hattrick style.  People can always change the style themselves with SkinPlugin or Stylus if they want to.

**Comment by rferromoreno** [comment]
> I assume this re-styling of the little badge by Foxtrick made sense at some point in the distant past. I had a very quick look at the code, and as far as I can see the re-styling does not convey any additional information to the user.
> 
> Personally, I can't see any reason why we shouldn't use the hattrick style. People can always change the style themselves with SkinPlugin or Stylus if they want to.

If you're OK with the change let me know, I can go ahead and remove what's no longer needed. Already did it in my local

**Comment by unnecessarydave** [comment]
Go for it.
